src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,87 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Contains interaction with user, meaning printing and reading through the scanner.
+ */
+public class Ui {
+    Scanner scanner;
+
+    Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    public void showLine() {
+        String line = "-----------------------------------";
```

Comment:
Can remove? just print straight. Also, missing javadoc header for these methods.